### ISSUE TITLE
Black/Shadowed thumbnails

### ISSUE THREAD

**Issue by Peabuttt** [jneilliii/OctoPrint-UltimakerFormatPackage#62]
I'm guessing the issue is probably between the chair and the computer, but here it goes... all my thumbnails are just like a shadow of the 3d model. check attachment. How can I fix it?
![Screenshot 2022-11-20 013701](https://user-images.githubusercontent.com/107125078/202878618-9af09b83-5195-4f38-b837-cf42f9397310.png)

**Comment by jneilliii** [comment]
The color of the thumbnail is based on the color of your filament profile. I can't control that unfortunately.

**Comment by Peabuttt** [comment]
So i googled and i also found that. I changed the color of the filament in cura just to test it (yellow). The result was the same. Maybe i changed something i thought it was the filament but it wasn't. 
![Screenshot 2022-11-20 024342](https://user-images.githubusercontent.com/107125078/202880223-7a2d5558-511e-4a4b-b38f-0f9153c96344.png)

[EDIT] Disregard what i just said. I tested again changing the color and it worked. Thank you for your time.
